docs/whatsnew/6.0.rst

```diff
@@ -210,8 +208,6 @@ with information about the parameter used when setting up the cosmology::
     >>> from astropy.cosmology import FlatLambdaCDM
     >>> FlatLambdaCDM.parameters["H0"]
     Parameter(derived=False, unit=Unit("km / (Mpc s)"), equivalencies=[], ...)
-    >>> FlatLambdaCDM.derived_parameters["Ode0"]
-    Parameter(derived=True, unit=None, equivalencies=[], ...)
```

Comment:
It might be better to keep the example with `"Ode0"` and to delete the one with `"H0"` so that this code block and the preceding one would access different parameters.

Reply:
Can you elaborate? ``Ode0`` is in `_derived_parameters`` which is private.

Reply:
The change log entry in the version I reviewed said that derived parameters should be accessible through `.parameters`, but the updated version does not state that anymore, so it is clear now that my suggestion is not valid.